src/lib.rs

```diff
@@ -354,3 +375,26 @@ pub fn xml_string_to_json(xml: String, config: &Config) -> Result<Value, Error>
     let root = Element::from_str(xml.as_str())?;
     Ok(xml_to_map(&root, config))
 }
+
+/// Returns a tuple for Array and Value enforcements for the current node or
+/// `(false, JsonArray::Infer(JsonType::Infer)` if the current path is not found
+/// in the list of paths with custom config.
+#[cfg(feature = "json_types")]
+#[inline]
```

Comment:
whats the thought for explicitly calling out inline here? From [the docs](https://doc.rust-lang.org/reference/attributes/codegen.html?highlight=inline#the-inline-attribute), it seems like the compiler will already inline where it feels is right

Reply:
It's a small function called from within a FOR LOOP. I thought it would be less stack pushing if it's inlined. Do you see a problem?

Reply:
no not necessarily, I've just never explicitly used `#[inline]` :laughing:  

since the attribute is a _suggestion_ to the compiler, and you have a good reason for the suggestion, I'm fine leaving it in